src/components/ListDetail/ListDetail.module.css

```diff
@@ -172,6 +192,33 @@
   }
 }
 
+/** List detail link-title 
+* 1) The text of the list item
+* 2) On viewports < lg, the text, along with its pseudo-bullet (if visible) should remain aligned to the left side of the list item, with the arrow aligned to the right. Giving the title and the pseudo-bullet a flex-grow of 1, while giving the arrow a flex-grow of 3, allows this
+*/
+
+.list-detail__link-title {
+  flex-grow: 1; /* 2 */
+
+  margin-right: var(--eds-size-1);
+}
+
+/**
+* List detail link-right
+* 1) A button containing an arrow icon that is only displayed on viewports < lg
+* 2) The button component is styled with a height of --eds-size-5 (40px) which would cause each list item to have extra vertical space. Display:contents removes the button's principal box and renders its contents (the Icon component) as normal. However due to a bug in browsers, this currently removes this button from the accessibility tree, meaning screen readers will ignore the button's contents. See https://developer.mozilla.org/en-US/docs/Web/CSS/display-box
+* 3) On viewports < lg, the arrow should remain aligned to the right side of the list item. Giving the arrow a flex-grow of 3, while giving the title and the pseudo-bullet a flex-grow of 1, allows this
+*/
+
+.list-detail__link-right {
+  display: none;
+
+  @media all and (max-width: $eds-bp-lg) {
+    display: contents; /* 2 */
```

Comment:
this is here because the Button component adds height to each list item, causing unwanted vert space between them